cmd/devstream/version.go

```diff
@@ -16,5 +16,5 @@ var versionCMD = &cobra.Command{
 }
 
 func versionCMDFunc(cmd *cobra.Command, args []string) {
-	fmt.Println(version.Version)
+	fmt.Println("Version:", version.Version)
```

Comment:
Do not add "Version:"; the reason is already explained in the previous comment.

Reply:
This version is for "dtm version",  and also this Version is used in other files instead of "plugin.version"

Reply:
yes,  got it